### ISSUE TITLE
Create a repeated operations I module quiz

### ISSUE THREAD

**Issue by mbcann01** [brad-cannell/intro.r.public#9]
# Overview

This was never created for the course. It should closely follow the Repeated Operations I lab.

**Comment by hahnjm** [comment]
**Task 1.** Please download **random_generated.csv**, import it into your R global environment as **df_abc** and view the data.
**Question 1.** The df_abc data frame contains [] rows and [] columns.
- General answer comments: The df_abc data frame contains 20 rows and 7 columns.
**Question 2.** Which function is suitable in the blank below to change the names of all columns to all lowercase letters?
`df_abc <- df_abc %>%
  [](str_to_lower)`
- solution: rename_with
**Question 3.** How many missing values are there in the albert column? 
- solution: 3
**Question 4.** Which column in the **df_abc** data frame contains the greatest number of missing values?
- solution: beth
**Question 5.** In order to calculate the mean for each column, what function should be used in the blank below?
`df_abc %>% 
  summarise(
    [](
      .cols = c(1:7),
      .fns  = ~ mean(.x, na.rm = TRUE)
    )
  )`

**Comment by mbcann01** [comment]
Hi @hahnjm, you are planning to add this to an Rmd file, correct?

**Comment by hahnjm** [comment]
> Hi @hahnjm, you are planning to add this to an Rmd file, correct?

Yes, I will convert this to an RMD file and place it in the shared space.
